packages/protocol/contracts-0.8/common/FeeCurrencyDirectory.sol

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../contracts/common/Initializable.sol";
+import "@openzeppelin/contracts8/access/Ownable.sol";
+
+interface IOracle {
+  function getExchangeRateFor(address identifier)
+    external
+    view
+    returns (uint256 numerator, uint256 denominator, uint256 lastUpdateTimestamp);
```

Comment:
given our last discussion, we agreed that we were not going to provide `lastUpdateTimestamp`?